### PR TITLE
Added feature to claim issues from github

### LINKED ISSUE #42: Claiming issues from Github
## Description
As a user
I need to claim issues from github
so that I can automate issue claiming.

### Update [Required]
- [ ] Add functionality to claim an issue from Github.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Added feature to claim issues via Github comment @sys-bot claim. This will only work if a person is a member of the organization. 
For more details read https://developer.github.com/v3/issues/assignees/#check-assignee, https://developer.github.com/v3/issues/assignees/#add-assignees-to-an-issue

Fixes #42 

# Type of Change:

- Code
- New feature (a non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
before_command
after_command

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by sammy1997** [comment]
@ramitsawhney27 You may have a look here https://developer.github.com/v3/issues/assignees/#check-assignee . The success code is 204.

**Comment by ramitsawhney27** [comment]
@sammy1997 you can resolve conflicts and then we can merge.

File: main_server.py [hunk 1]
```diff
@@ -38,6 +38,11 @@ def github_hook_receiver_function():
                 #If comment is to assign issue
                 elif comment_body.lower().startswith('@sys-bot assign') and len(tokens) == 3:
                     issue_assign(issue_number, repo_name, tokens[2], repo_owner)
+                elif comment_body.lower().startswith('@sys-bot claim') and len(tokens) == 2:
```
Comment (sara-02) [inline main_server.py]: we can do 
```
comment_body_str= comment_body.lower()
tokens_len= len(tokens)
```
And use these variables in the if-else case instead.

File: main_server.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
 from flask import Flask, jsonify
 from flask import request, json, Response
-from github_functions import label_opened_issue, issue_comment_approve_github, github_pull_request_label, issue_assign
+from github_functions import label_opened_issue, issue_comment_approve_github, github_pull_request_label, issue_assign, github_comment, issue_claim_github
```
Comment (sara-02) [inline main_server.py]: with multiple imports, it is good practice to use `()`
eg: `from github_functions import (label_opened_issue, issue_comment_approve_github, github_pull_request_label, issue_assign, github_comment, issue_claim_github)`

File: main_server.py [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 from stemming.porter2 import stem
 from nltk.tokenize import word_tokenize
 from auth_credentials import announcement_channel_id, BOT_ACCESS_TOKEN
 from slack_functions import dm_new_users, check_newcomer_requirements, approve_issue_label_slack, assign_issue_slack
```
Comment (sara-02) [inline main_server.py]: same as above ^^

File: messages.py [hunk 4]
```diff
@@ -18,5 +18,7 @@
     'correct_approve_format': 'The correct format is /sysbot_approve_issue <repo> <issue_no>',
     'error_slash_command': 'The slash command got a 500 error. Please check and try again.',
     'not_a_maintainer': 'You are not a maintainer. Please contact <@U0BKKUBQU|may> to get added.',
-    'correct_assign_format': 'The correct format is /sysbot_assign_issue <repo_name> <issue_no> <assignee_github_username>'
+    'correct_assign_format': 'The correct format is /sysbot_assign_issue <repo_name> <issue_no> <assignee_github_username>',
+    'not_an_org_member': 'You can\'t be assigned as you are not a member of the org. Join slack for more info',
```
Comment (sara-02) [inline messages.py]: You missed a period at the end of the message.

File: messages.py [hunk 5]
```diff
@@ -18,5 +18,7 @@
...
+    'wrong_format_github': 'Wrong format of command'
```
Comment (sara-02) [inline messages.py]: You missed a period at the end of the message.

**Pull request merged**